moto/cloudwatch/models.py

```diff
@@ -44,18 +60,127 @@ def __init__(self, namespace, name, value, dimensions):
                                      'value']) for dimension in dimensions]
 
 
+class MetricDatumT(BaseModel):
+
+    def __init__(self, namespace, data):
+        self.namespace = namespace
+        self.data = data
+
+
+class MetricDatumv2(BaseModel):
+
+    def __init__(self, namespace, metricdata):
+        self.Namespace = namespace
```

Comment:
It's unusual to have object attributes be uppercased like this. In this situation I'd expect `Namespace` to be a hardcoded class name. It makes me think this is how the code works:
```
class ANamespace:
    pass
class MetricDatumv2(BaseModel)
    def __init__(self):
        print(self.Namespace)
MetricDatumv2.Namespace = ANamespace
MetricDatumv2()  # prints 'ANamespace'
```

So I recommend using lowercase values for this and all similar variables throughout.

Reply:
ya it's against PEP, @JackDanger perhaps should enable some PEP flake tests?

Reply:
The test suite runs `flake8 ./moto` before every run and I'm not sure why this isn't being caught. Any ideas? Yes, we should absolutely tighten the flake8 constraints for developers. Maybe even adding autopep8 to help out would be useful.

Reply:
just created https://github.com/spulec/moto/pull/1232